authentication/NavBar.js

```diff
@@ -30,7 +30,7 @@ export default function NavBar(props) {
             {height: SpacingXLarge}
           ]}
         >
-          <Text>{loadTranslations("login")}</Text>
+          <Text>{loadTranslations("login")} </Text>
```

Comment:
formatting issue, lets only include touched files

Reply:
this was a styling issue, the text log in and sign up wouldn't fully appear. it needs the space

Reply:
if its a styling issue maybe we need to add a style to the text element ?

Reply:
i found the issue. it seems that it is because i have font set to bold in my device settings. ill remove the space but this is definitely somethin we should look into in the future